components/definition/plots/NetPyNEInclude.js

```diff
@@ -0,0 +1,367 @@
+      switch(element.constructor.name) {
+          case 'Number':
+            out['gids'].push(element)
+            break;
+          case 'String':
+            element!='all'?out['groups'].push(element):out.exclusive='all'
+            break;
+          case 'Array':
+            if (element[1].constructor.name=='Number') {
+              out['popids'][element[0]] = [element[1]]
+            }
+            else {
+            	out['popids'][element[0]] = element[1]
+            }
+            break;
+          default:
+            break
+        }
+    });
+    return out
+  }
+  collectInfo = () =>{
+    Utils
+      .sendPythonMessage("netpyne_geppetto.getGIDs", [])
+      .then((response) => {
+        if (response) {
+          Utils
+            .sendPythonMessage(this.props.model)
+            .then((response2) => {
+              if (response2) {
+                var included = this.convertFromPython(response2)
+                var clone = Object.assign({}, response)
+                Object.keys(clone).forEach((key) => clone[key]=false)
+                this.setState({
+                  data: response,
+                  include: included,
+                  secondPopoverOpen: clone,
+                  label: this.whoIsIncluded(included, response)
+                })
+              }
+          })
+        }
+    })
+  }
+    
+  handleMainPopoverOpen = (open, preventDefault=false, target=false) => {
+    // This prevents ghost click -> 
+    if (preventDefault) {
+      preventDefault
+    }
+    // close all secondary popovers
+    var clone = Object.assign({}, this.state.secondPopoverOpen)
+    Object.keys(clone).forEach((key) => {clone[key] = false})
+    
+    if (this.XOR(open, this.state.mainPopoverOpen)) {
+      this.setState({
+        mainPopoverOpen: open, 
+        secondPopoverOpen: clone, 
+        anchorEl: target?target:this.state.anchorEl})
+    }
+    if (!open) {
+      this.sendToPython()
+      this.setState({label: this.whoIsIncluded(this.state.include, this.state.data)})
+    }
+  }
+  
+  handleSecondPopoverOpen = (name, open, preventDefault=false, target=false) => {
+    // This prevents ghost click -> 
+    if (preventDefault) {
+      preventDefault;
+    }
+    var clone = Object.assign({}, this.state.secondPopoverOpen)
+    Object.keys(clone).forEach((key) => {clone[key] = (key==name)?open:false})
+    
+    if (!this.checkEqual(clone, this.state.secondPopoverOpen)) {
+      this.setState({
+        secondPopoverOpen: clone,
+        anchorEl2: target?target:this.state.anchorEl,
+      });
+    }
+    else{
+    }
+  };
+  
+  closeSecondPopover = () => {
+    var clone = Object.assign({}, this.state.secondPopoverOpen)
+    Object.keys(clone).forEach((key) => {clone[key] = false})
+    if (!this.checkEqual(clone, this.state.secondPopoverOpen)) {
+      this.setState({
+        secondPopoverOpen: clone,
+      });
+    }
+    this.setState
+  }
+  
+  defaultMenues = () => {
```

Comment:
Plural of Menu is Menus. I had to search as well -> https://en.wiktionary.org/wiki/menus#English